src/report/report-list.html

```diff
@@ -25,6 +25,30 @@ <h2>{{'report.viewReports' | message}}</h2>
             </tbody>
         </table>
     </div>
+    <div class="openlmis-table-container">
+        <table>
+            <caption class="error" ng-show="!vm.shipmentReports.jasper.length && !vm.shipmentReports.superSet.length">
+                {{'report.noReports' | message}}
+            </caption>
+            <thead>
+            <tr>
+                <th>{{'report.shipmentReports' | message}}</th>
```

Comment:
Here a one time binding would be better to use. It can speed up the app performence:
{{:: 'report.shipmentReports' | message}}
It can be applied for every non dynamic binding

Reply:
I added double colons.